tasks/coverage/codegen_sourcemap.snap

```diff
@@ -1273,7 +1274,8 @@ Invalid Character `[`
 (499:32-500:4) " {\n   " --> (247:33-248:0) " {"
 (500:4-500:12) " switch " --> (248:0-248:12) "\n\t\t\t\tswitch "
 (500:12-500:17) "(type" --> (248:12-248:17) "(type"
-(500:17-501:6) ".$$typeof) {\n     " --> (248:17-249:0) ".$$typeof) {"
+(500:17-500:4) " switch (type" --> (248:17-248:27) ".$$typeof)"
+(500:4-501:6) " switch (type.$$typeof) {\n     " --> (248:27-249:0) " {"
```

Comment:
Here caused due to lost `SwitchStatement` body span and using `SwitchStatement` span